src/backend/marsha/core/serializers.py

```diff
@@ -527,3 +567,19 @@ def get_url(self, obj):
             )
 
         return url
+
+    def update(self, instance, validated_data):
+        """Force to remove extension if present in title field."""
+        if not validated_data.get("title"):
+            return super().update(instance, validated_data)
+
+        match = re.match(
+            r"^(?P<title>.*)(?P<extension>\.{extension:s})$".format(
+                extension=EXTENSION_REGEX
+            ),
+            validated_data.get("title"),
+        )
+        if match:
+            validated_data.update(title=match.group("title"))
```

Comment:
You should be able to do this with re.sub(pattern, replacement, string)

Reply:
Using the `validate_title` method, I don't think we should use `re.sub`, you will once my fixup committed.